www/caddy-custom/Makefile

```diff
@@ -4,7 +4,7 @@
 
 PORTNAME=	caddy-custom
 PORTVERSION=	${CADDY_VERSION}.${XCADDY_VERSION}
-PORTREVISION=	14
+PORTREVISION=	15
```

Comment:
This is a problematic change since we're taking this revision and when FreeBSD does bump it we need to play cat and mouse increasing it again.

Reply:
What would be my leverage then if the packet has to change? The caddy version number+the xcaddy version number is already set, I would only have Portrevision to bump, I think.

Reply:
The general rule is don't unless you really really really have to. There's potential in this port to to upstream something more flexible perhaps.

Reply:
Okay, in this case I dont really have to. I will change it back.